Makefile

```diff
@@ -79,6 +89,10 @@ set_env_class_stage:
 	$(eval export ENABLE_AUTO_DEPLOY=true)
 	$(eval export DISABLE_AUTODELETE=1)
 	$(eval export PAAS_CF_TAG_FILTER=stage-*)
+	$(eval export ROOT_DNS_ZONE=cloudpipeline.digital)
+	$(eval export APPS_ROOT_DNS_ZONE=cloudpipelineapps.digital)
+	$(eval export ENV_DNS_ZONE=${AWS_ACCOUNT}.${ROOT_DNS_ZONE})
```

Comment:
I explained below why it's called `ENV` but we can simplify this as well. I'm open minded about it.